src/ripple/app/ledger/impl/LedgerConsensusImp.cpp

```diff
@@ -0,0 +1,1876 @@
+//------------------------------------------------------------------------------
+/*
```

Comment:
Got it. I was looking at the commits separately, but thanks @scottschurr for enumerating those changes, since they can be easy to miss overall.